src/kasn1.rs

```diff
@@ -37,6 +37,18 @@ impl<'a> DerEncBigUint<'a> {
         }
     }
 
+    /// Skip possible leading zeroes that do not affect sign of the resulting integer
+    pub fn new_unpad(data: &'a [u8]) -> KResult<Self> {
```

Comment:
I do not think we should have a different new method.
It should be all done within new() because this is always relevant to create proper Strict DER encoding.

Reply:
Sure. I will move it to normal new. I was thinking if it could break something, but it is unlikely.